common/azure-common/src/main/java/com/azure/common/http/policy/HttpLoggingPolicy.java

```diff
@@ -104,7 +104,7 @@ public HttpLoggingPolicy(HttpLogDetailLevel detailLevel, boolean prettyPrintJSON
                         reqBodyLoggingMono = collectedBytes.flatMap(bytes -> {
                             String bodyString = new String(bytes, StandardCharsets.UTF_8);
                             bodyString = prettyPrintIfNeeded(logger, request.headers().value("Content-Type"), bodyString);
-                            log(logger, String.format("%s-byte body:\n%s", contentLength, bodyString));
+                            log(logger, String.format("%s-byte body:%n%s", contentLength, bodyString));
```

Comment:
This is incorrect - the `\n` is being used to indicate a new line should be printed before the `bodyString` value was printed.

Reply:
%n works the same as %n in format string. Correct me if I am wrong.

Reply:
Ah, I see! This is fine.